frontend/src/services/TimeService.js

```diff
@@ -0,0 +1,31 @@
+export const timeElapseSincePosted = (date) => {
+    let currentDate = new Date();
+    let timeFormat = "sec";
+    let timePosted = currentDate.getTime() - date.getTime();
+    timePosted = (timePosted / 1000).toFixed(0);
+
+    if(timePosted < 60){
+        timePosted = (timePosted).toFixed(0);
+    }
+    else if(timePosted < 3600){
+        //Display in minutes
+        timePosted = (timePosted/60).toFixed(0);
+        timeFormat = "min";
+    }
+    else if(timePosted <86400){
+        //Display in hours
+        timePosted = (timePosted/(60*60)).toFixed(0);
+        timeFormat = "h";
+    }
+    else if(timePosted < 1036800){
+        //Display in days
+        timePosted = (timePosted/(60*60*24)).toFixed(0);
+        timeFormat ="days";
+    }
+    else if(timePosted >= 1036800){
+        timePosted = (timePosted/(60*60*24*12)).toFixed(0);
+        timeFormat = "years";
+    }
+           
+    return timePosted + timeFormat;
```

Comment:
maybe add a space between time posted and format? otherwise if we want to keep them stuck together it would be better to go for a fully compact way and do `s` for seconds, `m` for minutes, `d` for days and `y` for years (and remove the `ago`)? i think it would look a little aesthetically weird if it said "5days ago"

Reply:
Yeah sure